scripts/spectrum-tokens.js

```diff
@@ -119,7 +119,9 @@ const processPackages = async (srcPath, index) => {
         `\\s*--spectrum-alias-${packageName}[^;]*;`,
         'g'
     );
-    for (const varsPath of await fg([varsPaths])) {
+    for (const varsPath of await fg([varsPaths], {
+        ignore: ['**/tokens/**/*.css'],
```

Comment:
This is the part that was breaking the Radios (after the Tokens update was made). We'd previously not been expecting component tokens in global files, and this was eating the ones added in this update.